keyboards/mwstudio/cover60/info.json

```diff
@@ -0,0 +1,109 @@
+{
+    "manufacturer": "MWStudio",
+    "keyboard_name": "Cover60",
+    "maintainer": "TW59420",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["B6", "A10", "A9", "A8", "B15", "B14", "B13", "A4", "A5", "A6", "A7", "B10", "B11", "A3" ],
+        "rows": ["A2", "B12", "B5", "B7", "B9" ]
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "ws2812": {
+        "pin": "B8"
+    },
+    "rgblight": {
+        "led_count": 19,
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x6001",
+        "vid": "0x7BA1",
+        "force_nkro": true
+    },
+    "layouts": {
```

Comment:
Looking at the PCB image provided in `readme.md`, this board has more than one key arrangement available. Would you be able to add additional key arrangements to `layouts`?